src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -137,4 +142,56 @@
         }
         return new ProjectName(trimmedName);
     }
+
+    /**
+     * Parses a String into a description
+     *
```

Comment:
Remove extra spacing in javadocs, as well as methods below.

Reply:
Add period after each sentence, and perhaps give a short description of the param.